buildpipeline/pipeline.json

```diff
@@ -112,7 +111,7 @@
         {
           "Name": "Core-Setup-OSX-BT",
           "Parameters": {
-            "PB_PortableBuildArg": "-PortableBuild=true",
+            "PB_PortableBuildArg": "-portable",
```

Comment:
Yes, this is incorrect after https://github.com/dotnet/core-setup/commit/64a0b92cb524427a620f9760a4f02b41444eb089 and should be reverted.